src/screens/AddSiteFormScreen.js

```diff
@@ -127,6 +129,84 @@ class AddSiteFormScreen extends Component {
         })
     }
 
+    countyOptions() {
+        return Object.keys(counties).map((key) => {
+            return <Picker.Item key={key} label={counties[key]} value={key}/>;
+        })
+    }
+
+    renderCountyOptions() {
+        const {labelStyle, formInputStyle, pickerStyle} = styles;
+        const {countyOption, currentUser} = this.props;
+
+        if(currentUser.isAdmin){
+            return (
+                <View>
+                    <Text style={labelStyle}>{county} {optional}</Text>
+                    <Picker
+                        style={[formInputStyle, pickerStyle]}
+                        selectedValue={countyOption}
+                        onValueChange={this.onUpdateCountyOption}
+                    >
+                        {this.countyOptions()}
+                    </Picker>
+                </View>
+            );
+        }
+    }
+
+    forestOptions() {
+        return Object.keys(forest_names).map((key) => {
+            return <Picker.Item key={key} label={forest_names[key]} value={key}/>;
+        })
+    }
+
+    renderForestOptions() {
+        const {labelStyle, formInputStyle, pickerStyle} = styles;
+        const {forestOption, currentUser} = this.props;
+
+        if(currentUser.isAdmin){
+            return (
+                <View>
+                    <Text style={labelStyle}>{forest} {optional}</Text>
+                    <Picker
+                        style={[formInputStyle, pickerStyle]}
+                        selectedValue={forestOption}
+                        onValueChange={this.onUpdateForestOption}
+                    >
+                        {this.forestOptions()}
+                    </Picker>
+                </View>
+            );
+        }
+    }
+
+    mvumOptions() {
+        return Object.keys(mvum_names).map((key) => {
+            return <Picker.Item key={key} label={mvum_names[key]} value={key}/>;
+        })
+    }
+
+    renderMVUMOptions() {
+        const {labelStyle, formInputStyle, pickerStyle} = styles;
+        const {mvumOption, currentUser} = this.props;
+
+        if(currentUser.isAdmin){
+            return (
+                <View>
+                    <Text style={labelStyle}>{mvum} {optional}</Text>
+                    <Picker
+                        style={[formInputStyle, pickerStyle]}
+                        selectedValue={mvumOption}
+                        onValueChange={this.onUpdateMVUMOption}
+                    >
+                        {this.mvumOptions()}
+                    </Picker>
+                </View>
+            );
+        }
+    }
```

Comment:
GH-93 -- Three straightforward pickers, but they check `isAdmin` on `currentUser` to see if they render or not.